src/DotNet.Consolidate/Services/JsonLogger.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+
+using DotNet.Consolidate.Models;
+
+namespace DotNet.Consolidate.Services
+{
+    public class JsonLogger : ILogger
```

Comment:
This class violates a single responsibility principle.
Moreover, the `Message` method prints the raw sting to the console, which is not obvious.

I'd suggest extracting the `WriteAnalysisResults` to a separate interface and class.

Reply:
I took what was there.  I can extract the interface and do that but you already have a Logger class that already broke the Single Responsibility by having a method that aliased Console.WriteLine along with a method that wrote the results.  

The logger logs items and it changes the way items are logged so that the standard output is the json and the status messages that are logged under the message call still get output but are placed in the error stream.  Makes it so the json is the output and parsed without having to ignore the message output.  Since you had the default logger, I reimplemented the interface to change how it works when it is json output.  I would have done the same if I was doing an xml exporter or other format.  Pretty much  a standard way to have status messages. Git does this and other cli.   Logger class was the defacto Outputter of the program.  I'll look into doing as you wish but  the class was pretty much a way to generate output so I created a new output generator to follow the requirements of 
- Only json in the output
- Keep existing status messages in output some way.  Hencem the standard/Error output streams

Reply:
![image](https://github.com/olsh/dotnet-consolidate/assets/3629489/eb72bfbf-1b63-47cf-85df-37d81a5ada57)
Because in Powershell it appended more but see how the error stream contains the message that was sent in Message method.